### ISSUE TITLE
Add support OS Sierra

### ISSUE THREAD

**Issue by jeevatkm** [jeevatkm/ReplyWithHeader#97]
Will do it.

**Comment by jeevatkm** [comment]
Added support in v5.2-beta

**Comment by jeevatkm** [comment]
Download Page: http://bit.ly/1bsYyhg

**Comment by jeevatkm** [comment]
Reopened for v5.2 main release preparation.

**Comment by jeevatkm** [comment]
Added support for `de-*`

```
else if ([identifier hasPrefix:@"de"])
        identifier = @"de"; // considering all de-* into one umberlla
```

**Comment by jeevatkm** [comment]
Referring locale support issue #98

**Comment by jeevatkm** [comment]
Done! will be released in v5.2 this weekend, for now updating v5.2-beta. you can download it right away.

**Comment by jeevatkm** [comment]
Closing it for v5.2 release.
